src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -297,7 +297,58 @@ describe('Test suite for BulkConceptsPage component', () => {
     wrapper.find('#add-button').simulate('click');
     jest.runAllTimers();
   });
-
+  it('should handle CIEL concepts with mappings', () => {
+    const newProps = {
+      ...props,
+      concepts: [concept4],
+      loading: false,
+      datatypes: ['text'],
+      classes: ['Diagnosis'],
+      preview: {
+        url: 'dsa',
+        display_name: 'asd',
+      },
+      questionAnswers: [concept4],
+      recursiveQuestionAnswers: [concept4],
+    };
+    const wrapper = mount(<Provider store={store}>
+      <Router>
+        <BulkConceptsPage {...newProps} />
+      </Router>
+    </Provider>);
+    wrapper.find('#add-button').simulate('click');
+    expect(
+      wrapper.instance().props.children.props.children.props.recursiveQuestionAnswers[0]
+        .mappings.length,
+    ).toEqual(1);
+    jest.runAllTimers();
+  });
+  it('should handle CIEL concepts without mappings', () => {
```

Comment:
Are the answers the same as mappings?

Reply:
Yes, answers are a subset of mappings. Another example of a mappings subset is `CONCEPT-SET`. A clear example of this can be seen in this [link](https://qa.openconceptlab.org/orgs/CIEL/sources/CIEL/concepts/1268/) and this [link](https://qa.openconceptlab.org/orgs/CIEL/sources/CIEL/concepts/1085/).
This particular PR handles the addition of all various types of mappings for a concept.